### PR TITLE
controller/restore-operator: create seed member 

### PR DESCRIPTION
**Pull request by fanminshi**
TODO: Manual test creation of seed member. Expect the seed member should start with a backup.

### CODE REVIEW

File: pkg/controller/restore-operator/sync.go [hunk 1]
```diff
@@ -62,10 +67,21 @@ func (r *Restore) processItem(key string) error {
...
 
 	er := obj.(*api.EtcdRestore)
-	r.clusterNames.Store(key, er.Spec.BackupSpec.ClusterName)
-	r.restoreCRs.Store(er.Spec.BackupSpec.ClusterName, er)
-	// TODO: create seed member.
-	return nil
+	clusterName := er.Spec.BackupSpec.ClusterName
+	r.clusterNames.Store(key, clusterName)
+	r.restoreCRs.Store(clusterName, er)
+	m := &etcdutil.Member{
```
Comment (hongchaodeng) [inline pkg/controller/restore-operator/sync.go]: Abstract into a method

Comment (fanminshi) [reply pkg/controller/restore-operator/sync.go]: done. ptal

**Comment by hongchaodeng** [comment]
code looks good

**Comment by fanminshi** [comment]
manual test:

Create Restore CR:
```yaml
apiVersion: "etcd.database.coreos.com/v1beta2"
kind: "EtcdRestore"
metadata:
  name: restore-s3-cluster-backup
spec:
  clusterSpec:
    size: 3
    version: "3.1.8"
  BackupSpec:
    clusterName: s3-cluster-backup
    storageType: S3
    s3:
      s3Bucket: jenkins-etcd-operator
      awsSecret: aws
      prefix: test-prefix
  ```
  
```sh
$ kubectl  logs -f s3-cluster-backup-0000
2017-10-24 20:08:47.380311 I | etcdmain: etcd Version: 3.1.8
2017-10-24 20:08:47.380456 I | etcdmain: Git SHA: d267ca9
2017-10-24 20:08:47.380469 I | etcdmain: Go Version: go1.7.5
2017-10-24 20:08:47.380476 I | etcdmain: Go OS/Arch: linux/amd64
2017-10-24 20:08:47.380484 I | etcdmain: setting maximum number of CPUs to 2, total number of available CPUs is 2
2017-10-24 20:08:47.380522 N | etcdmain: the server is already initialized as member before, starting as etcd member...
2017-10-24 20:08:47.380566 I | embed: listening for peers on http://0.0.0.0:2380
2017-10-24 20:08:47.380597 I | embed: listening for client requests on 0.0.0.0:2379
2017-10-24 20:08:47.381350 I | etcdserver: recovered store from snapshot at index 1
```

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**